core/templates/pages/exploration-editor-page/editor-tab/templates/modal-templates/customize-interaction-modal.component.spec.ts

```diff
@@ -240,6 +240,31 @@ describe('Customize Interaction Modal Component', () => {
     expect(component.isSaveInteractionButtonEnabled()).toBe(true);
   });
 
+  it('should enable Save Interaction button for End Exploration', () => {
+    spyOn(component, 'getTitle').and
+      .returnValue(AppConstants.END_EXPLORATION);
+
+    let className = 'oppia-exploration-id-input';
+
+    spyOn(
+      document, 'getElementsByClassName'
+    ).withArgs(className).and.returnValue(undefined);
+
+    expect(component.isSaveInteractionButtonEnabled()).toBe(true);
+  });
+
+  it('should Save Interaction button when input in add Exploration ID', () => {
+    spyOn(component, 'getTitle').and
+      .returnValue(AppConstants.END_EXPLORATION);
+
+    let inputField = document.createElement('input');
+    let inputValue = angular.element(inputField);
+
+    (inputValue as any).attr !== '';
+
+    expect(component.isSaveInteractionButtonEnabled()).toBe(true);
+  });
+
```

Comment:
Also add a case where the last input box is empty and the save button is disabled.

Reply:
please review this test. i have pushed the changes